src/algorithm/SCAN-Cuda.cpp

```diff
@@ -0,0 +1,116 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-21, Lawrence Livermore National Security, LLC
```

Comment:
```suggestion
// Copyright (c) 2017-22, Lawrence Livermore National Security, LLC
```
Please update the copyright year in the new files you added.